src/index.ts

```diff
@@ -7,11 +11,10 @@ import {
   lookupDomain,
 } from '@esri/hub-common';
 import { IContentSearchRequest } from '@esri/hub-search';
-import { Request, Response } from 'express';
-import * as _ from 'lodash';
+
 import { version } from '../package.json';
 import { getDataStreamDcatAp201 } from './dcat-ap';
-import * as config from 'config';
+import { requiredFields } from './dcat-ap/dcat-dataset';
```

Comment:
So the required fields stay the same across all requests?

Reply:
That's right. Right now the dependencies are not configurable and this is the comprehensive list.